dataset/oloris-location.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenLORIS-Location
+layout: info
+Edit: 2020-11-15
+toc: false
+commentable: false
+protected: true
+mathjax: true
+---
+
+
+
+# OpenLORIS-Location Dataset
+
+>(L)ifel(O)ng (R)obotic V(IS)ion (**OpenLORIS**) - Location Dataset (**OpenLORIS-Location**) is designed for accelerating the localization research for indoor localization, which contains variant illumination and viewpoints, severe dynamic elements, repetitive features and daily changed items.
+
+## Dataset download
+> Our dataset can be downloaded at [OpenLORIS-Location dataset](). The dataset is an open dataset, and we welcome contribution of data from third parties.  Please contact [Fei Qiao](mailto:qiaofei@tsinghua.edu.cn) for more details.
+> Our project using OpenLORIS-Location dataset for training can be found at [RaP-Net](https://github.com/ivipsourcecode/RaP-Net)
+Please see the details in our paper below. You can also open an issue [here](https://github.com/lifelong-robotic-vision/lifelong-slam/issues?q=is%3Aissue) for any questions.
```

Comment:
This repo is for issue tracking of the OpenLORIS-Scene datasets. For OpenLORIS-Location, perhaps users should create issues on the RaP-Net repo.